include/deal.II/grid/grid_tools_cache.h

```diff
@@ -219,9 +234,15 @@ namespace GridTools
       vertex_to_cell_centers;
 
     /**
-     * An rtree object covering the whole mesh.
+     * A collection of rtree objects covering the whole mesh.
+     *
+     * Each entry of the map is constructed from the function
+     * extract_rtree_level() applied to the rtree returned by the function
+     * get_locally_owned_cell_bounding_boxes_rtree(), with the specified level
+     * in input.
      */
-    mutable RTree<std::pair<BoundingBox<spacedim>, unsigned int>>
+    mutable std::map<unsigned int,
+                     RTree<std::pair<BoundingBox<spacedim>, unsigned int>>>
```

Comment:
same issue here -- what does `level` correspond to?

Reply:
to the argument passed to `extract_rtree_level()`